### PR TITLE
Update instructions

### PR DESCRIPTION
**Pull request by freeformz**
Update the instructions for maintainers.

Remove + .gitignore public.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,44 +54,12 @@ Hugo will output a url that you can paste on your browser to see the live websit
 
 ---
 
-## <a name="maintainers"></a> For Maintainers
+# Maintainers
 
-### After fetching a pull request, start at the instructions above to [run Hugo](#runhugo) and generate the static files into the subtree public folder.
+The website is hosted on [Netlify](https://app.netlify.com/sites/gobridge-dot-org) (depsite the name of the repository).
```
Comment (carlisia) [inline CONTRIBUTING.md]: Little misspelling?

Comment (freeformz) [inline CONTRIBUTING.md]: fixed.

**Comment by freeformz** [comment]
PS: at some point we may want to do away with the "source" branch and rename the repo too.

**Comment by carlisia** [comment]
👍  to merge.

You might go ahead and do that change whenever you want @freeformz. I'm glad we don't have to do these shenanigans anymore.

**Pull request merged**